docs/theories/sets-and-relations.rst

```diff
@@ -189,11 +189,11 @@ More details can be found in :cite:`MengRTB17`.
 |                      |                                              |                                                                                    |
 |                      |                                              | ``Term X = solver.mkConst(s, "X");``                                               |
 +----------------------+----------------------------------------------+------------------------------------------------------------------------------------+
-| Transpose            | ``(transpose X)``                            | ``Term t = solver.mkTerm(Kind::TRANSPOSE, X);``                                    |
+| Transpose            | ``(rel.transpose X)``                        | ``Term t = solver.mkTerm(Kind::REL_TRANSPOSE, X);``                                |
```

Comment:
I think we renamed to `RELATION_` not `REL_`.

Reply:
Right. Fixed.